File: 1047/1047 - Tempo de Jogo com Minutos.cs
using System;
class Program
{
    static void Main()
    {
        int hi, mi, hf, mf, th, tm;
        string[] values = Console.ReadLine().Split();
        hi = int.Parse(values[0]); mi = int.Parse(values[1]); hf =  int.Parse(values[2]); mf = int.Parse(values[3]);
        th = hf - hi;
        tm = mf - mi;
        
        if (tm < 0)
        {
            tm += 60;
            th -= 1;
        }
        if (tm == 0 && th <= 0)
        {
            th += 24;
        }
        if (th < 0)
        {
            th += 24;
        }
        Console.WriteLine($"O JOGO DUROU {th} HORA(S) E {tm} MINUTO(S)");
    }
}

File: 1047/1047 - Tempo de Jogo com Minutos.cpp
#include <iostream>
int main() {
    int hi, mi, hf, mf, th, tm;
    scanf("%d%*c%d%*c%d%*c%d", &hi, &mi, &hf, &mf);
    th = hf - hi;
    tm = mf - mi;
    if (tm < 0) {
        tm += 60;
        th -= 1;
    }
    if (tm == 0 && th <= 0) {
        th += 24;
    }
    if (th < 0) {
        th += 24;
    }
    printf("O JOGO DUROU %d HORA(S) E %d MINUTO(S)\n", th, tm);
    return 0;
}